src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -581,6 +581,10 @@ public enum Mixins implements IMixins {
                     .addExcludedMod(FALSETWEAKS).addTargetedMod(TargetedMod.VANILLA).setApplyIf(() -> true)
                     .addMixinClasses("minecraft.MixinGameSettings_ReduceRenderDistance")),
 
+    BETTER_MOD_LIST(new MixinBuilder("Better Mod List").setPhase(Phase.EARLY).setSide(Side.CLIENT)
+            .addMixinClasses("fml.MixinGuiModList", "fml.MixinGuiSlotModList", "fml.MixinGuiScrollingList")
+            .setApplyIf(() -> TweaksConfig.betterModList).addTargetedMod(TargetedMod.VANILLA)),
```

Comment:
Can you also check if `EnderCoreTransformerClient` exists? So if it does, the mixin will never be applied. I'd imagine something like this:
```java
setApplyIf(() -> TweaksConfig.betterModList && !classExists("com.enderio.core.common.transform.EnderCoreTransformerClient"))

...

private static boolean classExists(String name) {
    try {
        Class.forName(name);
        return true;
    } catch (ClassNotFoundException e) {
        return false;
    }
}
```
I don't know if `EnderCoreTransformerClient` is loaded early enough though, that needs to be tested.